crates/api/src/lib.rs

```diff
@@ -36,7 +36,7 @@ pub use crate::types::*;
 pub use crate::values::*;
 
 cfg_if::cfg_if! {
-    if #[cfg(unix)] {
+    if #[cfg(all(unix, not(target_os = "fuchsia")))] {
```

Comment:
To avoid the extra `not(...)` could this list fuchsia first with `// no extensions at this time`

Reply:
I really wish Fuchsia wasn't marked as a unix